h/static/styles/partials/_admin-navbar.scss

```diff
@@ -0,0 +1,96 @@
+/**
+ * The dark navbar shown at the top of `/admin` pages.
+ */
+
+.admin-navbar {
+  background-color: $grey-7;
+  color: white;
+  display: flex;
+  flex-direction: row;
+  font-size: $normal-font-size;
+  justify-content: center;
+  min-height: 50px;
+}
+
+.admin-navbar__container {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  padding-left: 15px;
+  padding-right: 15px;
+  flex-grow: 1;
+
+  // Width chosen to match Bootstrap's "container" class,
+  // which is used for the main content of the page.
+  max-width: 970px;
+}
+
+.admin-navbar__logo-link {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.admin-navbar__logo {
+  color: $brand;
+}
+
+.admin-navbar__tab-list {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding-left: 20px;
+
+  // Override bootstrap defaults.
+  margin: 0;
+}
+
+.admin-navbar__tab-item {
+  flex-shrink: 0;
+  color: white;
+  padding-left: 10px;
+  padding-right: 10px;
+  text-align: center;
+
+  // Give tabs reasonable padding above and below text, even when wrapped onto
+  // multiple lines.
+  min-height: 2em;
+
+  // Color for the tab's link.
+  --link-color: $grey-3;
```

Comment:
The admin console does not need to work in IE, so we can use CSS variables here for styling descendant elements based on the state of a parent element.